lib/entropy/server/interfaces/main.py

```diff
@@ -6112,7 +6112,8 @@ def _is_pkg_free(self, repository_id, pkg_licenses):
                 return False
         return True
 
-    def _package_injector(self, repository_id, package_files, inject = False):
```

Comment:
Stay with the non-PEP-8 way. Second, long arguments list after def is done differently in this file - see e.g. def missing_runtime_blahblahblah.

Reply:
Why not follow PEP8?

Reply:
Because rest of the file, and even project does not. For consistency.

Reply:
do you think that could be fine then create a pr to clean code and move all to PEP8?

Reply:
Certainly, but it should do only that and not introduce any other change. Well there was an attempt to that: #1 but it has been closed.

Anyway, if you do it, merging my PR #68 would be a nightmare (provided that it ever gets attention…).

Reply:
ok. ;)